draft-ietf-satp-core.md

```diff
@@ -746,6 +771,15 @@ The parameters of this message consists of the following:
 
 - hash_transfer_init_claims REQUIRED: Hash of the Transfer Initialization Claims
   in the Transfer Proposal message.
+  
+- proof_manifest REQUIRED: A collection of signed assertions that provide proof that
```

Comment:
I think this is a good idea; we (should) include the type of proof along with the proof payload as part of the receipt

Reply:
In draft core-04 the set of items that is assumed to be gained from Stage-0 is listed in Section 7.1 (Transfer Initialization Claims). 

The Stage-1 Commence message carries a hash of these agreed claims:
hash_transfer_init_claims REQUIRED: Hash of the Transfer Initialization Claims in the Transfer Proposal message.